tests/quizzes/good/all-basic-questions/quiz.json

```diff
@@ -72,6 +72,12 @@
             "questions": [
                 "../../../questions/good/tf-true"
             ]
+        },
+        {
+            "name":"a-image-question",
+            "questions":[
+                "../../../questions/good/prompt-image"
```

Comment:
This quiz is about just having basic questions, one for each question type.
This question breaks the pattern.

I would just create a new quiz and have just this question in it.
You can call is something like "image-quiz".